### ISSUE TITLE
respository s3

### ISSUE THREAD

**Issue by jurgenweber** [elastic/elasticsearch-cloud-aws#94]
I am trying to get s3 repository configuration working and I must admit I am finding the documentation hard to follow, firstly my configuration file I have added:

```
repositories:
  s3:
    base_path: production
    bucket: searchidx
    region: eu-west-1
  type: s3
```

what else do I need? Looking in the debug log nothing is happening and my bucket is empty (yes I created the bucket)

**Comment by johnrengelman** [comment]
Hey @jurgenweber did you ever figure this out? I'm having the same problem. The repository doesn't ever get created, but I can't find any better documentation.

**Comment by jurgenweber** [comment]
I abandoned it, this was a while ago. I think I worked out nothing just turns up their but you then have to send curl commands to the esearch instance to take a snapshot, etc. I hope i'm right here.

**Comment by tlrx** [comment]
@johnrengelman @jurgenweber To use S3 repository, you basically need to:
* install the [elasticsearch-cloud-aws](https://github.com/elastic/elasticsearch-cloud-aws) plugin on every node of the cluster
* add your S3 access keys in the `elasticsearch.yml` file like documented [here](https://github.com/elastic/elasticsearch-cloud-aws#generic-configuration), also on every node
* create a S3 repository with a tool like `curl` and then create/restore snapshots, see the doc [here](http://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html)

Note that access keys can be configured directly when creatins/updating the S3 repository.

Hope this helps.
